### DISCUSSION TITLE
Discussion about last rejected photos / Обсуждение отклоненных фотографий

### DISCUSSION THREAD

**Discussion by Duche3d** [PastVu/pastvu#44] [Ideas / Идеи и предложения]
Должна быть страница с отклоненными фотографиями, где пользователи могли бы их обсуждать и убеждать модераторов в необходимости их публикации. Фотографии могут жить на этой странице ограниченное время по прошествии которого, если они так и не будит опубликованы одним из модераторов, они окончательно удаляются. Дабы туда не попадало всякое порно - администраторы должны иметь возможность чистить этот раздел по своему усмотрению;
Приоритет этой задачи - низкий

**Comment by anvaruk** [comment]
я бы добавил , 

страница как раздел у пользователя, видимый  только пользователю и модераторам, куда будут собираться отклоненные фотографии. (может даже всем залогиненым пользователям)

там будет скапливаться определенное количество 20-50 фоток, старые удаляются по мере поступления новых

автор (а может даже все залогиненые) и модераторы могут их коментировать, владелец может отправить ее на премодерацию повторно (после устранения ошибок), но только 1 - 2 раза например
удалять отсюда может только модератор (если совсем уж порно допустим) остальные нужно чтоб всегда висели чтобы видеть на случай если пользователь постоянно пытается одну и туже картинку пропихнуть, ну и если делать видимым всем, то пусть обсуждают, просто ее на карте не будет

окончательный вариант хорошо бы обсудить с модераторами

**Comment by Duche3d** [comment]
Какой-то механизм указания причины отклонения нужен уже сейчас. Посему переводим это задачу в неотложные.

**Comment by paul-k-pastvu** [comment]
Тут нужно как-то переформулировать задачу с учетом текущих реалий.
Связано с задачей организации процедуры апелляции.

**Comment by paul-k-pastvu** [comment]
Итого. Все фотографии, даже отклоненные, остаются на сайте навечно.
При  этом уже есть вот такая задача - #171 - как раз про то, что для отклоненных и отправленных на доработку фото тоже нужна возможность комментирования.

Но это всё не ведет нас ни какой внятной процедуре апелляции.
